sdcm/gemini_thread.py

```diff
@@ -60,28 +60,38 @@ def run(self):
 
 class GeminiStressThread():  # pylint: disable=too-many-instance-attributes
 
+    gemini_result_file_name = "gemini_result.log"
+
     def __init__(self, test_cluster, oracle_cluster, loaders, gemini_cmd, timeout=None, outputdir=None, params=None):  # pylint: disable=too-many-arguments
         self.loaders = loaders
         self.gemini_cmd = gemini_cmd
         self.test_cluster = test_cluster
         self.oracle_cluster = oracle_cluster
         self.timeout = timeout
         self.result_futures = []
-        self.gemini_log = None
         self.outputdir = outputdir
         self.gemini_commands = []
+        self._gemini_result_file = None
         self.params = params if params else {}
 
-    def _generate_gemini_command(self, loader_idx):
+    @property
+    def gemini_result_file(self):
+        if not self._gemini_result_file:
+            result = self.loaders.nodes[0].remoter.run("echo $HOME", ignore_status=True)
```

Comment:
this is again repeating the logic of getting the $HOME directory, yet again.
and fallback to `/home/centos` is a problem, also to run in GCE with other type of machines.
we have manger jobs running monitor on ubuntu and debian.

and also there an ongoing work, for running on amazon linux, and the centos user would be only an alias, and this could would break.

Reply:
Moved to LoaderSet logic to get the gemini_base_path